docs/source/en/_toctree.yml

```diff
@@ -253,6 +253,8 @@
         title: GPT Neo
       - local: model_doc/gpt_neox
         title: GPT NeoX
+      - local: model_doc/gpt_neox_japanese
```

Comment:
You should also add the model to the main README.

Reply:
I wrote README, README_ko, README_zh-hans and README_zh-hant.